HelpSource/Classes/ScopeView.schelp

```diff
@@ -74,54 +85,54 @@ EXAMPLES::
 SUBSECTION:: A step-by-step example
 
 code::
-// execute these in succession
-(
-s.boot;
-)
+s.boot; // boot server (if not already booted), then execute the following two blocks in succession:
+
 (
 f = Buffer.alloc(s,1024,2);
 b = Bus.audio(s,1);
 
 w=Window.new.front;
+w.onClose = { // free everything when finished
+  c.stop; a.free; d.free; f.free; b.free;
+  "SynthDefs, busses and buffers have all been freed.".postln;
+};
 c = ScopeView(w.view,w.view.bounds.insetAll(10,10,10,10));
 c.bufnum = f.bufnum;
+c.server_(s); // Important: one must assign the ScopeView to a server
 )
+
 (
-// listening to the bus, using ScopeOut to write it to the buffer
+// listening to the bus, using ScopeOut2 to write it to the buffer
 a=SynthDef("monoscope", { arg bus, bufnum;
-                var z;
-                z = In.ar(bus,2);
-                // ScopeOut writes the audio to the buffer
-                ScopeOut.ar(z, bufnum);
-            }).play(
-                RootNode(s),
-                [\bus,b.index, \bufnum, f.bufnum] ,
-                \addToTail // make sure it goes after what you are scoping
-            );
+  var z;
+  z = In.ar(bus, 2);
+  // ScopeOut2 writes the audio to the buffer (It only works with ScopeOut2 for some reason, not with ScopeOut, so this was corrected)
```

Comment:
You can take out the section in parenthesis. There's something wrong with `ScopeOut`. Better not to mention it.